pkg/endpoint/bpf.go

```diff
@@ -274,14 +274,6 @@ func (e *Endpoint) writeHeaderfile(prefix string, owner Owner) error {
 	// Endpoint options
 	fw.WriteString(e.Opts.GetFmtList())
 
-	fw.WriteString("#define LXC_PORT_MAPPINGS ")
```

Comment:
out of curiosity how did you decide this was unused? it's written here unconditionally and the one reference is predicated on it being defined, as opposed to populated (https://github.com/cilium/cilium/blob/master/bpf/bpf_lxc.c#L77). That `if` also checks `!DISABLE_PORT_MAP` so that's probably what led you here.

Reply:
Well, given that PortMap was never used, I presume that this line was always empty, so it also corresponds to dead code in the BPF datapath (which @tgraf correctly pointed out we should remove as well). I'll clean this stuff up :)